submissions/Harnytskyi/friends-app/js/script.js

```diff
@@ -0,0 +1,151 @@
+const GRID = document.getElementById("allFriends");
+const CHECKBOX_MAN = document.getElementById("gender-male");
+const CHECKBOX_WOMAN = document.getElementById("gender-female");
+const SORT_AZ = document.getElementById("a-z");
+const SORT_ZA = document.getElementById("z-a");
+const SEARCH_INPUT = document.getElementById('search');
+const SORT_09 = document.getElementById("0-9");
+const SORT_90 = document.getElementById("9-0");
```

Comment:
I also can suggest `SORT_60` because of the speed limit and `SORT_88` just because the cool number.

Reply:
renamed `SORT_90` to `SORT_AGE_DESCEND`